syntax/cloud.php

```diff
@@ -85,7 +85,6 @@ public function handle($match, $state, $pos, Doku_Handler $handler)
      */
     public function render($mode, Doku_Renderer $renderer, $data)
     {
-        if ($mode != 'xhtml') return false;
```

Comment:
This is potentially problematic, the mode might be something we don't understand. Like a XML renderer, or a something else. We can only do this if we never manipulate `$this->doc` ourselves but always call parent renderer methods - I am not sure this is the case?

(this applies to all adjusted render() methods.)

Reply:
Alright, then instead of deleting, changing to this maybe resolve potential issue?

``if ($mode != 'xhtml' and $mode != 'metadata') return false;``